addons/google-analytics/README.md

```diff
@@ -20,9 +26,83 @@ module.exports = {
 }
 ```
 
-Then, set an environment variable
+## Basic configuration
 
+The analytics addon can be configured using story parameters with the `analytics` key.
+To configure globally, import `addParameters` from your app layer in your `preview.js` file.
+
+```js
+import { addParameters } from '@storybook/client-api';
+
+addParameters({
+  analytics: {
+    reactGAId: 'UA-000000-01',
+    reactGAOptions: {},
+  },
+});
 ```
+
+Or alternatively define the parameters as global variables in `main.js`
+
+```js
 window.STORYBOOK_GA_ID = UA-000000-01
 window.STORYBOOK_REACT_GA_OPTIONS = {}
 ```
+
+## Advanced configuration
+
+The analytics addon gives complete control over what `Storybook events` are logged to Google Analytics.
+This can be configured using the `events` parameter, which is a map of key-value pairs denoting what storybook events should be tracked and under what circumstances.
+
+
+### Excluding events from Google Analytics tracking
+
+Excluding events from Google analytics tracking, is as simple as assigning a `false` value to the corresponding storybook event.
+
+```js
+import { addParameters } from '@storybook/client-api';
+import { STORY_CHANGED } from '@storybook/core-events';
+
+addParameters({
+  analytics: {
+    reactGAId: 'UA-000000-01',
+    reactGAOptions: {},
+    events: {
```

Comment:
not sure if `events` is a very descriptive configuration parameter for defining what events we want to track.

I'm open to suggestions on this.